### ISSUE TITLE
Is it possible a month is negative or out of 1..=12?

### ISSUE THREAD

**Issue by tisonkun** [BurntSushi/jiff#130]
Hi @BurntSushi.

I noticed that `month` method is defined as:

```rust
    #[inline]
    pub fn month(&self) -> i8 {
        self.date().month()
    }
```

I wonder if it's possible month is negative or out of `1..=12`?

**Comment by BurntSushi** [comment]
No, it isn't possible. We should add that guarantee to the docs. And for other similar methods like `day`.

But it is mentioned in the constructor for `civil::Date`: https://docs.rs/jiff/latest/jiff/civil/struct.Date.html#method.new
